launch/tier4_perception_launch/launch/perception.launch.xml

```diff
@@ -83,13 +83,18 @@
     default="$(var data_path)/traffic_light_fine_detector"
     description="options: `tlr_yolox_s_batch_**`. The batch number must be either one of 1, 4, 6"
   />
-  <arg name="traffic_light_fine_detector_model_name" default="tlr_yolox_s_batch_6" description="options: `tlr_yolox_s_batch_**`. The batch number must be either one of 1, 4, 6"/>
+  <arg name="traffic_light_fine_detector_model_name" default="tlr_pedcar_yolox_s_batch_6" description="options: `tlr_pedcar_yolox_s_batch_**`. The batch number must be either one of 1, 4, 6"/>
```

Comment:
```suggestion
  <!-- cspell: ignore pedcar -->
  <arg name="traffic_light_fine_detector_model_name" default="tlr_pedcar_yolox_s_batch_6" description="options: `tlr_pedcar_yolox_s_batch_**`. The batch number must be either one of 1, 4, 6"/>
```

Reply:
Thanks for the correction. We decided to change the model's name in https://github.com/autowarefoundation/autoware.universe/pull/5871/commits/25f879bd1e0eb99aa2ed6ff04adfbb724b040267